db/mongodb_test.go

```diff
@@ -106,6 +107,25 @@ func TestMongoDB(t *testing.T) {
 	if err := db.CreateExtraIndexes([]string{"teste.index1"}); err == nil {
 		t.Error("expected errors running extra indexes, got nil")
 	}
+	q := Query{Uf: "SP", Limit: 1}
+	_, err = db.Search(q)
+	if err != nil {
+		t.Errorf("expected no error getting query, got %s", err)
+	}
+	q = Query{Uf: "SP", Limit: 1}
+	sr, _ := db.Search(q)
+	var r map[string]interface{}
+	if err := json.Unmarshal([]byte(sr), &r); err != nil {
+		panic(err)
+	}
+	d := r["data"].([]interface{})
+	if len(d) > 0 {
```

Comment:
Isso tem que fazer parte do teste. É esperado um resultado e se `len(d)` não for um precisamos de um falha específica nos testes, algo como `if len(d) !=  1 { t.Errorf(…) }`.